src/actions/loopring/index.js

```diff
@@ -15,8 +15,84 @@ import { getDepth } from "../../lightcone/api/v1/depth/get";
 import { getMarketInfo } from "../../lightcone/api/v1/marketinfo/get";
 import config from "../../lightcone/config";
 import { getBalances } from "../../lightcone/api/v1/balances";
-import { fromWei } from "web3-utils";
 import { getFeeRates } from "../../lightcone/api/v1/fee-rates";
+import { weiToEther } from "../../utils";
+import { getEthNonce } from "../../lightcone/api/v1/ethnonce/get";
+import { getRecommendedGasPrice } from "../../lightcone/api/v1/recommendedGasPrice/get";
+
+// auth status
+
+export const GET_AUTH_STATUS_SUCCESS = "GET_AUTH_STATUS_SUCCESS";
+export const GET_AUTH_STATUS_START = "GET_AUTH_STATUS_START";
+export const GET_AUTH_STATUS_END = "GET_AUTH_STATUS_END";
+
+export const getAuthStatus = (address) => async (dispatch) => {
+    try {
+        dispatch({ type: GET_AUTH_STATUS_START });
+        let needsRegistration = false;
+        try {
+            await lightconeGetAccount(address);
+        } catch (error) {
+            needsRegistration = true;
+        }
+        dispatch({ type: GET_AUTH_STATUS_SUCCESS, needsRegistration });
+    } catch (error) {
+        toast.error(<FormattedMessage id="error.auth.status" />);
+        console.error("error checking auth status", error);
+    } finally {
+        dispatch({ type: GET_AUTH_STATUS_END });
+    }
+};
+
+// registration
+
+export const register = (web3Instance, ethereumAccount) => async (dispatch) => {
```

Comment:
Could you test on testnet and attach the transaction please?

Reply:
Goerli example [here](https://goerli.etherscan.io/tx/0x465b39f23fdf5f9cf96009b04a057967a580a83859783f279c5f6c21414db6c9)